sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 4.1.0-preview (Unreleased)
```

Comment:
Note: when ready for release, you'll need to add the date here yourself. The pipeline script will do it too late, I'm told.

Reply:
I've been adding this myself on the recent releases. I'll keep that in mind!